kong/plugins/aws-lambda/schema.lua

```diff
@@ -14,5 +14,6 @@ return {
     log_type = {type = "string", required = true, default = "Tail", 
                        enum = {"Tail", "None"}},
     port = { type = "number", default = 443 },
+    unhandled_response = {type = "number", default = -1, required = true},
```

Comment:
> Would something in the 5xx range be a healthier default? 

I was using `-1` as the value so I could short circuit if the flag was unset. What happens to `numeric` values when unset?

> As a second note on that line, wouldn't unhandled_status be a more appropriate name?

Yep, I'll change it and was conflicted on what to use as the name myself. 👌